spring-integration-amqp/src/main/java/org/springframework/integration/amqp/outbound/AbstractAmqpOutboundEndpoint.java

```diff
@@ -193,10 +194,22 @@ public void setLazyConnect(boolean lazyConnect) {
 	}
 
 	/**
+	 * Set the value to set in the {@code x-delay} header when using the
+	 * RabbitMQ delayed message exchange plugin. By default, the {@link AmqpHeaders#DELAY}
+	 * header (if present) is mapped; setting the delay here overrides that value.
+	 * @param delayExpression the expression.
+	 * @since 5.0
+	 */
+	public void setDelay(int delay) {
+		this.delayExpression = new SupplierExpression<>(() -> delay);
```

Comment:
I don't think it's necessary to add it to the XML since a literal numeric looks the same; with expressions that evaluate to String, you need quotes:

     delay-expression="12"

Vs.

    foo-expression="'bar'"